docs/components/movement-sensor/adxl345.md

```diff
@@ -100,37 +143,39 @@ Edit and fill in the attributes as applicable.
 ## Attributes
 
 <!-- prettier-ignore -->
-| Name                  | Type   | Inclusion    | Description                                                                                                                                                                                    |
-| --------------------- | ------ | ------------ | ----------------- |
-| `board`               | string | **Required** | The `name` of the [board](/components/board/) to which the device is wired.                                                                                                                    |
-| `i2c_bus`             | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device.                                                            |
+| Name                  | Type   | Inclusion    | Description |
+| --------------------- | ------ | ------------ | ----------- |
+| `board`               | string | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
+| `i2c_bus`             | string | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on the [board](/components/board/) wired to this device. |
 | `use_alt_i2c_address` | bool   | Optional     | Depends on whether you wire SDO low (leaving the default address of 0x53) or high (making the address 0x1D). If high, set true. If low, set false or omit the attribute. <br> Default: `false` |
-| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [table below](#tap-attributes).                                                                |
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [table below](#freefall-attributes).                                                     |
+| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [table below](#tap-attributes). |
```

Comment:
```suggestion
| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [Tap attributes](#tap-attributes). |
```

What do you think of a more explicit link title here? Same with other `[table below]` instances.

Reply:
sure